mantle/harness/testresult/status.go

```diff
@@ -16,6 +16,7 @@ package testresult
 
 const (
 	Fail TestResult = "FAIL"
+	Warn TestResult = "WARN"
 	Skip TestResult = "SKIP"
 	Pass TestResult = "PASS"
```

Comment:
random comment: In the future I think it would be cool if we could use colors for these outputs. I'm thinking red, yellow, white, green.

Reply:
what's about `s/white/blue/` ?

Reply:
👍

Reply:
Did this work for you? I can't get colors to output locally.

Reply:
For me that works. tmux and gnome-terminal have no problems

Reply:
not working for me, but I can try to figure that out later.

Reply:
ok I got it working :)

Reply:
what was wrong?

Reply:
Another example is NetworkManager. `nmcli c show` will show colors, but `nmcli c show | cat` won't because the output is to a pipe.

Reply:
> what was wrong?

I didn't have the code checked out right locally :(